extractor/src/main/java/org/schabi/newpipe/extractor/services/media_ccc/extractors/MediaCCCParsingHelper.java

```diff
@@ -28,9 +28,9 @@ public static OffsetDateTime parseDateFrom(final String textualUploadDate) throw
         }
     }
 
-    public static boolean isLiveStreamId(final String url) {
+    public static boolean isLiveStreamId(final String id) {
         final String pattern = "\\w+/\\w+";
```

Comment:
This pattern should be compiled and made `static final`

Reply:
Done